ember-app/app/components/hb-milestone-card.js

```diff
@@ -1,7 +1,43 @@
 import HbCard from "../components/hb-task-card";
+import Ember from "ember";
+
+var { get, set } = Ember;
+
+var takeWhile = function(list, callback, context) {
```

Comment:
Does [`_.takeWhile()`](https://lodash.com/docs#takeWhile) not work in this case?